src/app/features/fss-search/fss-search-results/fss-search-results.component.ts

```diff
@@ -21,15 +22,17 @@ export class FssSearchResultsComponent implements OnChanges {
   ngOnChanges(): void {
     this.searchResultVM = [];
     if (this.searchResult.length > 0) {
+      let currentPage = this.currentPage; 
       var batches = this.searchResult[0];
-      for (var i = 0; i < batches.length; i++) {
+      for (var i = 0, SrNo = 1; i < batches.length; i++, SrNo++) {
```

Comment:
Please change the capitalisation of SrNo to camel-case to match the convention used for other variables

Reply:
updated as suggested. Thanks